diff --git a/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/control b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/control
--- a/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/control
+++ b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/control
@@ -10,7 +10,7 @@ Vcs-Browser: https://github.com/chriszero/tvspielfilm2xmltv
 
 Package: vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv
 Architecture: all
-Depends: python (>= 2.7), python-requests, python-tz, ${misc:Depends}
+Depends: python2.7, python-requests, python-tz, ${misc:Depends}
 Description: TV Spielfilm Grabber for the xmltv2vdr vdr plugin
  A Grabber for the xmltv2vdr vdr plugin that uses TV Spielfilm as data source.
  .
diff --git a/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/python2.7 b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/python2.7
new file mode 100644
--- /dev/null
+++ b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/python2.7
@@ -0,0 +1,8 @@
+--- vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.5.orig/tvspielfilm2xmltv.py
++++ vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.5/tvspielfilm2xmltv.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python
++#!/usr/bin/python2.7
+ # encoding: utf-8
+ '''
+ setup -- shortdesc
diff --git a/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/series b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/series
new file mode 100644
--- /dev/null
+++ b/vdr-plugin-xmltv2vdr-grabber-tvspielfilm2xmltv-0.0.0/debian/patches/series
@@ -0,0 +1 @@
+python2.7
-- 
1.7.2.5

